.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Tell us about a problem you are experiencing
+labels: Bug
+
+---
+
+**What steps did you take and what happened:**
+[A clear and concise description of what the bug is, and what commands you ran.]
+
+
+**What did you expect to happen:**
+
+
+**The output of the following commands and cstor pod logs will help us better understand what's going on**:
+(Pasting long output into a [GitHub gist](https://gist.github.com) or other [Pastebin](https://pastebin.com/) is fine.)
+
+* `kubectl logs deployment/maya-apiserver -n openebs`
+* `kubectl logs deployment/<pv-name>-target -n <openebs_namespace> -c cstor-istgt`
+* `kubectl get pods -n openebs`
```

Comment:
IMO good to get the output of pool configurations
`kubectl get csp -l openebs.io/storage-pool-claim=<spc_name> -o yaml`